public/templates/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link crossorigin="anonymous" href="./fixed-data-table.min.css" media="all" rel="stylesheet">
```

Comment:
CSS may be included via Webpack, see _e.g._ https://webpack.github.io/docs/stylesheets.html